src/decentralized-borrowing.yaml

```
@@ -12,24 +12,503 @@ publishStatus: Live
 socialShareImage:
 showIncorrectOnCompletion: true
 steps:
-  - uuid: c82bf74b-d0a0-444c-b09c-35f3017ec6e5
-    name: What is decentralized borrowing
+  - uuid: 30274820-c5f6-47ab-a61f-4a74e1047aad
+    name: Important terms
     content: |
-      What is decentralized borrowing
+      Liquidation:
+      The selling of collateral to cover the debt is called liquidation.
```

```suggestion
      - **Liquidation** : The selling of collateral to cover the debt is called liquidation.
```
We can show all the terms as bullet points with the headings as bold.